### PR TITLE
fix a few issues in documentation

### PR DESCRIPTION
**Pull request by mcornaton**
- fix "Application I/O" anchor;
  - add reference to `IEXEC_INPUT_FILES_NUMBER` variable in relevant section;
  - use the same formatting for all "Congratulations" texts (bold);
  - fix typos

### CODE REVIEW

File: for-developers/your-first-app.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In this guide, we will prepare an iExec app based on an existing docker image an
 **Tutorial Steps :**
 
 * [Understand what is an iExec decentralized application?](your-first-app.md#understand-what-is-an-iexec-decentralized-application)
-* [Application I/O](your-first-app.md#application-i-o)
```
Comment (PierreJeanjacquot) [inline for-developers/your-first-app.md]: gitbook needs the `-` in this link to work properly, see:
- https://documentations.iex.ec/staging/v/fix-typos/for-developers/your-first-app#application-i-o is working
- https://documentations.iex.ec/staging/v/fix-typos/for-developers/your-first-app#application-io is NOT working

revert this change

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
please fix link (see comment)

**Comment by mcornaton** [comment]
IntellIj was highlighting this particular link as non-working but now I'll know! :)

**Review by PierreJeanjacquot — APPROVED** [review]
LGTM

**Comment by PierreJeanjacquot** [comment]
Thank you for contributing @mcornaton :tada: 
Feel free to merge and delete the branch :)

**Comment by mcornaton** [comment]
That should be alright. Let me know if I forgot anything!
